### DISCUSSION TITLE
Releases UI

### DISCUSSION THREAD

**Discussion by techguy16** [community/community#6512] [Other Feature Feedback, Questions, & Ideas]
It's a great improvement to the UI, but I would like to see an indicator beside the downloads with a little symbol displaying what Operating System it it for. But I already like the concept of a more modern interface.

**Comment by MylesBorins** [comment]
@techguy16 we are thinking through a variety of improvements we can make to artifacts / downloads including improving the experience of highlighting the downloads for the platform you are visiting GitHub from.

This work is not currently scoped, nor do we have a date to ship it... but it is on our radar.

**Reply by techguy16** [reply]
Is this work in development yet?

**Comment by techguy16** [comment]
My UI Design (with the icons)

![Proposed UI](https://user-images.githubusercontent.com/88870951/198500572-8ebedf4c-3196-4283-97eb-d5e0bc1e1b65.png)
